adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+from gzip import BadGzipFile
+import random
+from collections import Counter
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    bag_of_letters = []
+    letter_bank = []
+    # based on the value add the letter n times to alphabet list
+    for letter,value in LETTER_POOL.items():
+        for i in range(value):
+            bag_of_letters.append(letter)
+
+    while len(letter_bank) != 10:
+        draw = random.choice(bag_of_letters)
+        letter_bank.append(draw)
+        bag_of_letters.remove(draw)
+    
+    return letter_bank
```

Comment:
Nice job completing your first pair project. I noticed you are using a for loop to create the bag of letters and appending the letter from the LETTER_POOL based on the value of the letter. I assume you are using this as a weight but you could also use the second argument in random.choice so what's the letter bank is weighted based on that argument. I also noticed you removed from the bag of letters once you got the random letter but instead of creating a new data structure you could do a check for if the letter exists in the LETTER_POOL and if it does exist, append it to the letter bank and decrease the value of that letter by one: `LETTER_POOL[draw] -= 1`